src/acceptanceTest/java/uk/gov/hmcts/futurehearings/hmi/acceptance/hearings/PUTHearingsValidationTest.java

```diff
@@ -93,11 +93,11 @@ void test_update_hearing_with_CFT_destination_system_without_body() throws Excep
         commonDelegate.test_expected_response_for_supplied_header(
                 getAuthorizationToken(),
                 getRelativeURL(), "hearing-request-without-body.json",
-                createPayloadWithCFTDestinationHeader(),
+                createPayloadWithCFTDestinationHeader("SNL", "CFT"),
                 null,
                 getUrlParams(),
                 getHttpMethod(),
                 HttpStatus.BAD_REQUEST, "hearings",
-                getHmiSuccessVerifier(),"Bad Request.",null);
+                getHmiSuccessVerifier(),"",null);
```

Comment:
Should this be the HMIErrorVerifier...I think there was a Cut Copy Paster error...